vars/main.yml

```diff
@@ -50,3 +52,12 @@ fw_services:
   zabbix:
     filter_INPUT:
       - dport: 10050
+
+  # For hosts running containerized services, but MySQL on the host, we need
+  # to permit connections from "everywhere that is not the primary interface"
+  # (because we don't know in advance the source ip or even the cidr range of
+  # the containerized services).
+  mysql_local:
+    filter_INPUT_raw:
+      - >-
+        -p tcp --dport 3306 ! -i {{ fw_primary_interface }}
```

Comment:
does it need a `-j ACCEPT` ?

Reply:
I guess not, you added it here: https://github.com/CCI-MOC/ansible-role-moc-firewall/commit/7260ac89e9f84b7f5894e5908bc7927e5c9d750e

Reply:
No, actually, that was a good catch! Because this is a "raw" rule it's used pretty much verbatim. We do need the explicit ACCEPT here (fixed).